{
    "URI.MalformedURIException.html": {
        "modified": "2021-07-24 18:49:52.778073",
        "size": "14.31KB",
        "md5": "136ac4c0f8db46e8a9f7fe9aea35bb9f",
        "sha256": "cccd0c7d8e1e9aa76ec95d52d45f6de134289b905210c00538a2a2c9ca7391ab"
    },
    "Metadata": {
        "Content-Encoding": "ISO-8859-1",
        "Content-Type": "text/html; charset=ISO-8859-1",
        "X-Parsed-By": [
            "org.apache.tika.parser.DefaultParser",
            "org.apache.tika.parser.html.HtmlParser"
        ],
        "X-TIKA:content_handler": "ToTextContentHandler",
        "X-TIKA:embedded_depth": "0",
        "X-TIKA:parse_time_millis": "2",
        "dc:title": "Uses of Class org.apache.axis.types.URI.MalformedURIException (Axis API)",
        "resourceName": "b'URI.MalformedURIException.html'",
        "title": "Uses of Class org.apache.axis.types.URI.MalformedURIException (Axis API)"
    }
}